04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+We wanted to highlight the importance of prevention and incentivise islands to contribute to the Common Pool, thus if the amount of resources in the common pool exceeds the threshold when the disaster happens, then the “power” of disaster towards the islands and common pool will be decreased. It makes sense in real life, as more preparation will result in less damage. The actual damage of disaster is decreased by half, but this can vary later.\\
+
+\subsection{Example distribution}
+This can be described by the examples in table X, which shows 4 possible scenario:\\ 
+
+\begin{itemize}
+    \item Contribution to common pool does not meet threshold and common pool cannot fully mitigate disaster’s damage
+    \item Contribution to common pool does not meet threshold but common pool can fully mitigate disaster’s damage
+    \item Contribution to common pool meets threshold but common pool cannot fully mitigate disaster’s damage
+    \item Contribution to common pool meets threshold and common pool can fully mitigate disaster’s damage
+\end{itemize}
+
+\begin{table}[h]
+\begin{center} 
+\begin{tabular}{|p{0.9in}|p{0.9in}|p{0.9in}|p{0.9in}|p{0.9in}|} \hline
+& \textbf{Scenario 1} & \textbf{Scenario 2} & \textbf{Scenario 3} & \textbf{Scenario 4} \\ \hline
+\textbf{Disaster total damage} & 1000 & 200 & 1200 & 1200 \\ \hline
+\textbf{CP's threshold} & 500 & 500 & 500 & 500 \\ \hline
+\textbf{CP's current value} & 300 & 300 & 500 & 700 \\ \hline
+\textbf{CP's multiplier} & 1 & 1 & 0.5 & 0.5 \\ \hline
+\textbf{Mitigated damage} & 300 & 200 & 500 & 1200 \\ \hline
+\textbf{Remaining damage} & 700 & 0 & 100 & 0 \\ \hline
+\textbf{CP's remaining value} & 0 & 100 & 0 & 100 \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Demonstration of how common works in different scenarios}
+\end{center} 
+\end{table}
+
+In scenario 1, the disaster total damage is high, however there is no preparation from the islands through contribution to the common pool. As a result, the common pool could not mitigate all of the damage and 70\% of the damage will be directly on the islands\\
+
+In scenario 2, even though there is no preparation, the common pool is still able to mitigate all of the damage thanks to the low damage of disaster. The leftover resource in the common pool will stay there.\\
+
+In scenario 3, the disaster damage is high. Fortunately, the islands have prepared themselves moderately through contribution to the common pool. Thus, the islands only have to take very little leftover damage.\\
+
+In scenario 4, the disaster damage is high and the islands have prepared themselves very well. As a result, the damage is fully mitigated by the common pool and there is still leftover resource in the pool. 
```

Comment:
comma mistake - should be a comma before 'and'

-- check the guidelines

Reply:
same with unnecessary \\ at the end - does not belong at the end of every paragraph